tap_quickbooks/schemas/time_activities.json

```diff
@@ -149,6 +149,12 @@
       ],
       "format": "singer.decimal"
     },
+    "ProjectRef": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
```

Comment:
As per this doc: https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/timeactivity#timeactivity
timeactivity api seems to have same schema as "shared/ref_schema.json"